python-sdk/tests_integration/databases/test_all_databases.py

```diff
@@ -56,7 +56,7 @@
         },
     ],
     indirect=True,
-    ids=["bigquery", "postgres", "redshift", "snowflake", "sqlite", "delta", "duckdb"],
+    ids=["bigquery", "postgres", "snowflake", "sqlite", "delta", "duckdb"],
```

Comment:
An alternative to remove redshift/comment it out would be to use
```
-k "not redshift"
```

and having a redshift marker:
https://docs.pytest.org/en/7.1.x/example/markers.html#marking-test-functions-and-selecting-them-for-a-run

Reply:
will checkout that out and try it.

Reply:
Added the `-k "not redshift"` for integration test cases.